src/loadable.js

```diff
@@ -90,7 +90,10 @@ function loadable(
       return <LoadingComponent {...this.props} />
     }
   }
-
+  // development mode load every time
+  if(process.env.NODE_ENV === 'development'){
+    LoadableComponent.load()
```

Comment:
1. This code should be executed only of the second call of this function.
2. This code could be executed each time in node.js environment.